doc/release/v1_4_0.md

```diff
@@ -11,6 +11,9 @@ Important Changes
 * Fixed old typo, the keyword `suit` has been
   replaced with `suite`, maintaining the backward
   compatibility.
+* Change logic for RTF_*_IF(condition, message) 
+  functions, adding new functions RTF_*_IF_TRUE
+  and RTF_*_IF_FALSE, deprecating the old ones 
```

Comment:
The Markdown of this comment is not rendering correctly. I suspect you should make sure that the asterisk is not interpreted as the beginning of an italic sentence, for example using back-ticks: *not good*, `*good*`.

Reply:
Furthermore, do not forget the point at the end of the sentence.